backend/docker-compose.yml

```diff
@@ -1,3 +1,6 @@
+# Easy startup of Tracim backend services needed for testing with pytest
+# Also useful for local development
```

Comment:
I not seen configuration for elasticsearch or elasticsearch-ingest, is it normal ?

Reply:
Maybe this tool can be added in docker-compose-opt.yml file ?

Reply:
There is an elasticsearch service in the file. ingest isn't used in the automated tests so I didn't include it for now.